src/Components/WebAssembly/testassets/StandaloneApp/Pages/FetchData.razor

```diff
@@ -52,8 +52,11 @@ else
     protected override async Task OnParametersSetAsync()
     {
         startDate = StartDate.GetValueOrDefault(DateTime.Now);
-        forecasts = await Http.GetJsonAsync<WeatherForecast[]>(
-            $"sample-data/weather.json?date={startDate.ToString("yyyy-MM-dd")}");
+        var request = new HttpRequestMessage(HttpMethod.Get, $"sample-data/weather.json?date={startDate.ToString("yyyy-MM-dd")}");
+        request.WithRequestCache(RequestCache.NoCache);
```

Comment:
Example of what this looks like to configure on a per-request basis

Reply:
This looks good, but can we avoid putting random new behaviors into `(repo)/src/Components/WebAssembly/testassets/StandaloneApp`? The "real" test asset projects are the ones in `(repo)/test/testassets`.

I know that `(repo)/src/Components/WebAssembly/testassets/StandaloneApp` is a misnomer. It's intended as the equivalent to MvcSandbox, but we had to call it "testassets" at some point in the past due to repo naming convention rules. I think those rules are no longer applicable so we could consider renaming it to `(repo)/src/Components/WebAssembly/samples/StandaloneApp` at some point.